src/Test1.java

```diff
@@ -1,10 +1,10 @@
 public class Test1 {
 
     public static void main(String[] args) {
-        System.out.println("local test 1");
```

Comment:
Why did you remove existing test1?

Reply:
ah sorry missed this one let me roll back

Reply:
@TalhaOzleblebici Updated

Reply:
Thank you